var/www/html/db-config.php.ctmpl

```diff
@@ -126,328 +128,3 @@ $wpdb->max_connections = 10;
  * Default: true
  */
 $wpdb->check_tcp_responsiveness = true;
-
-/** Configuration Functions **/
```

Comment:
I'd prefer preserving these comments in this file, as it's the best available documentation for how to use HyperDB.